cli/src/dap_client.rs

```diff
@@ -0,0 +1,88 @@
+use std::str::FromStr;
+use url::Url;
+
+use janus_client;
+
+use janus_messages::{Duration, TaskId};
+use prio::vdaf::prio3::Prio3Histogram;
+
+use crate::{CliResult, DetermineAccountId, Error, Output};
+use clap::Subcommand;
+use divviup_client::{self, Histogram};
+
+#[derive(Subcommand, Debug)]
+pub enum DapClientAction {
+    /// create a new task for the target account
+    Upload {
+        #[arg(long)]
+        task_id: String,
+        #[arg(long)]
+        value: usize,
```

Comment:
I suspect to accomodate for sumvec we'd need this to be a string, so we can select multiple buckets `1, 2, 3`. Or perhaps this can be a `Vec<usize>` with clap's [num_args](https://docs.rs/clap/latest/clap/struct.Arg.html#method.num_args) or [value_delimiter](https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_delimiter). This otherwise looks like a fine representation for a measurement.